skuld/openstackmap.py

```diff
@@ -591,6 +670,8 @@ def preload_regions(self, regions=None, all_regions_but=None):
                 all_regions.remove(self.osclients.region)
                 regions = list(all_regions)
                 regions.append(self.osclients.region)
+            else:
```

Comment:
I do not understand the semantic of this operation from if not regions to regions = list(all_regions) could you add some comments here?